src/components/diagrams/networkAreaDiagram/network-area-diagram-content.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+import React, { forwardRef } from 'react';
+import PropTypes from 'prop-types';
+import clsx from 'clsx';
+import { RunningStatus } from '../../util/running-status';
+import makeStyles from '@mui/styles/makeStyles';
+import { commonDiagramStyle } from '../diagram-common';
+
+const useStyles = makeStyles((theme) => ({
+    ...commonDiagramStyle(theme),
+    divNad: {
+        '& svg': {
```

Comment:
this CSS rule is in this `NadDiagramStyle` and in `commonSldStyle` then why not moving it in `commonDiagramStyle` for all ?